loky/backend/context.py

```diff
@@ -18,9 +18,15 @@
 import multiprocessing as mp
 from multiprocessing import get_context as mp_get_context
 from multiprocessing.context import BaseContext
+from concurrent.futures.process import _MAX_WINDOWS_WORKERS
```

Comment:
I think it's slightly safer to hard code `61` here too, just in case Python decides to rename the variable or change the location of `_MAX_WINDOWS_WORKERS`?

The numbers 60 or 61 are mentioned a few times in comments, but does not appear in the code. The only way to figure out the number's origin is to look at [CPython's source](https://github.com/python/cpython/blob/73250000ac7d6a5e41917e8bcea7234444cbde78/Lib/concurrent/futures/process.py#L114) or click one of the referenced links.